job-dsls/jobs/kie_jenkinsScripts_PR.groovy

```diff
@@ -84,13 +84,14 @@ job(jobName) {
 
     triggers {
         githubPullRequest {
+            useGitHubHooks()
+            userWhitelist(["appformer", "kiegroup"])
```

Comment:
userWhiteList? I thought that orgWhitelist is enough?

Reply:
@mareknovotny we tested it and for some reason it is required by the hook. It didn't work without it. I would like to know the exact reason why it's happening but I don't, I will try to investigate it and will let you know as soon as I have something.

Reply:
ok

Reply:
please add there a comment about this why it is there as we could forget and remove it accidentally

Reply:
@mareknovotny let me make a final test once jenkins is back just to be sure I'm not wrong with this

Reply:
So I was confused, the userWhiteList is not needed

Reply:
removed